pkg/server/server.go

```diff
@@ -440,6 +452,11 @@ func (s *Server) IsExperimentallyEnabled(flag ExperimentalFeatureFlag) bool {
 	return slices.Contains(s.experimentals, flag)
 }
 
+// IsAccessControlEnabled returns true if the access control feature is enabled.
+func (s *Server) IsAccessControlEnabled() bool {
+	return s.IsExperimentallyEnabled(ExperimentalAccessControlParams) && s.AccessControl.Enabled
```

Comment:
Sorry, it just occurred to me, why do we need an experiment _and_ a flag?? A flag should be enough